beta/src/components/ExternalLink.tsx

```diff
@@ -7,12 +7,11 @@ import * as React from 'react';
 export function ExternalLink({
   href,
   target,
-  children,
   ...props
 }: JSX.IntrinsicElements['a']) {
   return (
     <a href={href} target={target ?? '_blank'} rel="noopener" {...props}>
-      {children}
+      {props.children}
```

Comment:
What is this meant to fix? Why?

Reply:
<img width="741" alt="Screenshot 2021-11-12 at 10 13 14 PM" src="https://user-images.githubusercontent.com/32865581/141502959-6698e5c1-a7df-4e4c-ae22-b0818db7f5fc.png">

Reply:
But the original code _is_ doing that. So why was the original code flagged?

Reply:
Maybe rename `props` to something like `rest` and it'll work.

Reply:
Sorry I missed it. Nice catch Thanks 🏅